profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp

```diff
@@ -155,6 +155,7 @@ bool profile_grouped_conv_fwd_impl(int do_verification,
 
         if(op_ptr->IsSupportedArgument(argument_ptr.get()))
         {
+            std::cout << op_ptr->GetTypeString() << " supports this problem" << std::endl;
```

Comment:
I suppose it is not needed. We print perf for supported instance

Reply:
will delete